controllers/virtualmachineservice/virtualmachineservice_controller.go

```diff
@@ -392,6 +393,11 @@ func (r *ReconcileVirtualMachineService) createOrUpdateService(ctx *context.Virt
 		service.Spec.ExternalName = vmService.Spec.ExternalName
 		service.Spec.LoadBalancerIP = vmService.Spec.LoadBalancerIP
 		service.Spec.LoadBalancerSourceRanges = vmService.Spec.LoadBalancerSourceRanges
+		if service.Spec.Type == corev1.ServiceTypeLoadBalancer {
```

Comment:
Hmm.  It loos like the nodeports already allocated for the service (before this change) will not get cleaned up automatically (from https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation).  How do we handle that?  Do we have a release note (or some other way) to tell customers to clean up those nodeports manually?  Do customers even have RBAC perms to delete those nodeports?